parts/k8s/cloud-init/artifacts/cse_install.sh

```diff
@@ -68,7 +68,7 @@ installDeps() {
     fi
   fi
 }
-installGPUDrivers() {
+downloadGPUDrivers() {
```

Comment:
Renamed this func to clear up persistent confusion about what is accomplished here. Nothing is actually installed (and we invoke this for *all* VM SKUs, i.e., during VHD CI, so it's repeatedly confusing that we think we're installing GPU drivers in contexts without GPU).